c2corg_ui/static/partials/mapswitch.html

```diff
@@ -0,0 +1,5 @@
+<button class="btn orange-btn btn-default no-list-btn hidden-xs card-list" ng-click="MapswitchCtrl.toggle()" >
+  <span class="glyphicon glyphicon-globe"></span>
+  <span translate ng-show="!MapswitchCtrl.hideMap">Hide Map</span>
```

Comment:
Should be ``Hide map`` instead of ``Hide Map`` if you want to reuse the existing translation.